src/main/java/org/testng/reporters/JUnitReportReporter.java

```diff
@@ -185,6 +178,50 @@ public void generateReport(List<XmlSuite> xmlSuites, List<ISuite> suites,
 
   }
 
+  private TestTag createIgnoredTestTagFor(ITestNGMethod method) {
```

Comment:
@juherr - I didn't make this as a static method because no other methods in this class are static. So I thought its better to leave this method as an instance method as well.